charts/ci-operator/templates/steps/standard.yaml

```diff
@@ -86,6 +86,300 @@ step:
   script: |
     gitleaks -v --pretty --repo-path $(resources.inputs.source.path) --commit-from=$$(params.commitFrom) --commit-to=$(params.commitBefore) --branch=$(params.branch) --report gitleaks-report.json
 ---
+apiVersion: ci.w6d.io/v1alpha1
+kind: Step
+metadata:
+  name: git-leak-standard
+  labels:
+  {{- include "ci-operator.labels" . | nindent 4 }}
+  annotations:
+    ci.w6d.io/kind: generic
+    ci.w6d.io/task: git-leak
+    ci.w6d.io/order: "0"
+step:
+  name: git-leak-standard
+  image: "zricethezav/gitleaks:latest"
+  script: |
+    mkdir -p $(workspaces.artifacts.path)/tests/report-gitleaks
+    gitleaks --repo-path $(resources.inputs.source.path) -v --report=$(workspaces.artifacts.path)/tests/report-gitleaks/report-pathsecretsscan.json
+---
+apiVersion: ci.w6d.io/v1alpha1
+kind: Step
+metadata:
+  name: code-cov-standard
+  labels:
+  {{- include "ci-operator.labels" . | nindent 4 }}
+  annotations:
+    ci.w6d.io/kind: generic
+    ci.w6d.io/task: code-cov
+    ci.w6d.io/order: "0"
+params:
+  - name: CODECOV_URL
+    description: define the codecov url
+    type: string
+    default: "https://codecov.io/bash"
+  - name: CODECOV_TOKEN
+    description: define by client defines the token (found by the user)
+    type: string
+step:
+  name: code-cov-standard
+  image: "bash"
+  script: |
+    mkdir -p $(workspaces.artifacts.path)/tests/report-codecov
+    echo y | apk add --no-cache curl
```

Comment:
as running in non root not available in CI

Reply:
done